common/remote/rmtssh.cpp

```diff
@@ -521,9 +523,22 @@ class CFRunSSH: public CInterface, implements IFRunSSH
                 if (res.length()==0)
                     PROGLOG("%d: %s(%d): [OK]",n+1,slaves.item(n),reply.item(n));
                 else if (strchr(res.str(),'\n')==NULL)
+                {
                     PROGLOG("%d: %s(%d): %s",n+1,slaves.item(n),reply.item(n),res.str());
+                    if (reply.item(n)){
+                        errCode = reply.item(n);
```

Comment:
Trivial: inconsistent brace formatting in method/changes. 
My rule is, if small keep it consistent with exiting formatting, if larger change, reformat at least the method you are altering to our preferred brace formatting conventions, i.e. brace on newline.